src/ResourceManager/Resources/Commands.Resources/packages.config

```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.Azure.Management.Authorization" version="2.8.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.ManagementGroups" version="1.2.0-preview" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.ResourceManager" version="1.6.0-preview" targetFramework="net452" />
 </packages>
```

Comment:
You will need to remove the LocalFeed package before this can be merged.  Once the nuget package is published, remove this package and let me know, and I will remove the "Do Not Merge" tag.

Reply:
Yes. The SDK PR will be done today and I will remove it from here.